lib/tasks/fake_data.rake

```diff
@@ -216,11 +216,60 @@ namespace :fake_data do
     end
   end
 
+  desc 'create fake journeys'
+  task create_journeys: :environment do
+    puts 'create_journeys...'
+    Move
+        .left_outer_joins(:journeys).where(journeys: { move_id: nil })
```

Comment:
some strange indentation here, wonder how rubocop didn't pick this up 🤔

Reply:
maybe 4 indents to signify line continuation vs 2 indents for being inside a block?